pyani_plus/methods/method_anib.py

```diff
@@ -121,7 +120,7 @@ def parse_blastn_file(blastn: Path) -> tuple[float | None, int | None, int | Non
                     f"Found {len(fields)} columns in {blastn}, not {len(BLAST_COLUMNS)}"
                 )
                 raise ValueError(msg)
-            query = fields[0]
+            query = fields[BLAST_COL_QUERY]
```

Comment:
This was implicitly assuming the query was the first column (column index zero).